allium/lib/relays.py

```
@@ -698,16 +702,21 @@ def _aggregate_total_data_to_groups(self):
-        from .bandwidth_formatter import format_data_volume_with_unit, compute_total_data_pct, pick_best_period, _BEST_PERIOD_ORDER
+        from .bandwidth_formatter import (
+            BEST_PERIOD_ORDER,
+            compute_total_data_pct,
+            format_data_volume_with_unit,
+            pick_best_period,
+        )
         relays = self.json["relays"]
         net_by_period = self.json.get('network_health', {}).get('network_total_data_by_period', {})
         
         for category in self.json.get("sorted", {}):
             for key, group_data in self.json["sorted"][category].items():
```

Ruff (0.16.0)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)